portfolio/src/main/webapp/script.js

```diff
@@ -11,19 +11,81 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+const imgFilePath = '/images/album/';
```

Comment:
nit for general style; if you leave out the / at the end here, then the file paths below are more readable. No need to change.